client/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "files.insertFinalNewline": true
+}
```

Comment:
missing final new line

Reply:
added a final new line.
Is prettier run across all the files including .files?

Reply:
It works on .js and .json files. You can customise it to use single quote instead of double quote as well. (I prefer single quote but not a huge deal either way)